proto/cosmos/evm/erc20/v1/erc20.proto

```diff
@@ -32,6 +32,27 @@ message TokenPair {
   Owner contract_owner = 4;
 }
 
+// Allowance is a token allowance only for erc20 precompile
+message Allowance {
+  option (gogoproto.equal) = false;
+
+  // erc20_address is the hex address of ERC20 contract
+  string erc20_address = 1;
```

Comment:
```suggestion
  // address is the hex address of ERC20 contract
  string address = 1;
```

I think you can skip `erc20_` since it was added to the proto of ERC20.